index.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
     "scss/no-duplicate-dollar-variables": true,
     "at-rule-blacklist": null,
     "at-rule-empty-line-before": ["always", {
-      "except": ["after-same-name"]
+      "ignoreAtRules": ["function", "if", "each", "include", "mixin", "for", "else", "import", "media"]
```

Comment:
I can't find in the docs most of this options for `ignoreAtRules`: https://stylelint.io/user-guide/rules/at-rule-empty-line-before

Reply:
https://stylelint.io/user-guide/rules/at-rule-empty-line-before#ignoreatrules-array-of-at-rules